Dockerfile

```
@@ -0,0 +1,19 @@
+# The following does not work in Podman unless you build in Docker
+# compatibility mode: <https://github.com/containers/podman/issues/8477>
+# You can manually prepend every RUN script with `set -ex` too.
+SHELL ["sh", "-exc"]
+
+# Security-conscious organizations should package/review uv themselves.
+COPY --from=ghcr.io/astral-sh/uv:latest /uv /usr/local/bin/uv
+
+# set up a virtual env to use for whatever app is destined for this container.
+# RUN uv venv --python 3.12.5 /venv
+
+COPY . .
```

Hadolint

[warning] `COPY` to a relative destination without `WORKDIR` set.

(DL3045)